.travis.yml

```diff
@@ -35,3 +39,10 @@ script:
   - make install-deps-ui
   - make lint-ui
   - make build-ui
+
+deploy:
+  provider: script
+  script: make docker-push
+  on:
+    type: push
+    condition: $TRAVIS_BRANCH =~ ^(develop|master)$
```

Comment:
I would probably follow the `skywire-services` standard here and use the `branch` field as well as the condition to only push if it is not a PR. 

This would also eliminate some code in the Makefile that we otherwise dont need. And the Makefile is already pretty cluttered imo.

Reply:
ok, will do so. Thanks